### PR TITLE
feat(api): add italian messages

### LINKED ISSUE #42: [api] add italian conversation to slack_api_connection
In addition, update the tests

### PR DESCRIPTION
**Pull request by kevinpcf**
* add italian messages for slack_api_connection file
* update integration tests

Closes #42

### CODE REVIEW

File: src/api/slack_api_connection.py
Comment (alfcan) [inline src/api/slack_api_connection.py:1]: Need to add request language detection when handling requests coming to the bot.
* Add detect_language method of LanguageHandler class in handle_request method of slack_api_connection

File: tests/api_test/slack_api_connection_integration_test.py [hunk 1]
```diff
@@ -20,10 +24,85 @@ def test_handle_request(self, mocker):
                      "actions": [{"action_id": "action-yes"}],
                      "message": {"ts": 10}}
 
+        self.language_handler.detect_language("Text in English")
```
Comment (alfcan) [inline tests/api_test/slack_api_connection_integration_test.py:27]: Maybe this is not necessary

File: tests/api_test/slack_api_connection_integration_test.py [hunk 2]
```diff
@@ -20,10 +24,85 @@ def test_handle_request(self, mocker):
...
+        mocker.patch('intent_handling.tools.os.environ.get', side_effect=[
+            "CADOCSNLU_URL_PREDICT", "0.55"])
+
+        expected_response = {"message": "true"}
+        response = slack_api_connection.handle_request(payload)
+
+        # Assertion
+        assert expected_response == response
+
+    def test_handle_request_it(self, mocker):
```
Comment (alfcan) [inline tests/api_test/slack_api_connection_integration_test.py:93]: You can parameterize the previous test that checks the same integrations by passing the payload text as a parameter

**Review by alfcan — CHANGES_REQUESTED** [review]
Check the comments in the review

File: src/api/slack_api_connection.py
Comment (kevinpcf) [reply src/api/slack_api_connection.py:1]: I'll fix in next commit

File: tests/api_test/slack_api_connection_integration_test.py (hunk 1 above)
Comment (kevinpcf) [reply tests/api_test/slack_api_connection_integration_test.py]: I'll fix in next commit

File: tests/api_test/slack_api_connection_integration_test.py (hunk 2 above)
Comment (kevinpcf) [reply tests/api_test/slack_api_connection_integration_test.py]: I'll fix in next commit

File: tests/api_test/slack_api_connection_integration_test.py [hunk 3]
```diff
@@ -20,10 +24,85 @@ def test_handle_request(self, mocker):
...
+
+        data_test = {"channel": {"id": 1},
+                     "user": {"id": "id_test"},
+                     "actions": [{"action_id": "action-yes"}],
+                     "message": {"ts": 10}}
+
+        self.language_handler.detect_language("Testo in Italiano")
```
[the remark this answers is not in the document]
Comment (kevinpcf) [reply tests/api_test/slack_api_connection_integration_test.py]: I'll fix in next commit

**Review by alfcan — APPROVED** [review]

**Pull request merged**